package.json

```diff
@@ -31,6 +30,7 @@
     "@sentry/types": "^5.17.0",
     "expo-constants": "*",
     "expo-device": "*",
+    "expo-updates": "^0.2.4",
```

Comment:
Maybe this might be an issue maybe not. But if `expo-updates@0.3.x` is released, it won't be included in this range. You can test it [with this tool](https://semver.npmjs.com/). We might want to use something like `>=0.2.4` instead.

What happens is all `0.x.x` versions are considered pre-release and beta/alpha versions. Because of this, a minor bump is considered a breaking change. It's a small minor but important detail of the SemVer spec.

Reply:
ah good catch

since 0.3.0 _might_ be a breaking change then, is it better to keep this at 0.2 and just update once the next minor version of `expo-updates` releases? it's a small chance, but wouldn't want it to break apps